frontend/src/metabase/css/core/colors.module.css

```diff
@@ -32,6 +33,32 @@
   --mb-color-text-primary: var(--mb-color-text-dark);
   --mb-color-text-secondary: var(--mb-color-text-medium);
   --mb-color-text-tertiary: var(--mb-color-text-light);
+
+  /**
+    * Base colors
```

Comment:
Based on [this discussion.](https://metaboat.slack.com/archives/C063Q3F1HPF/p1721049128029239?thread_ts=1721029953.539449&cid=C063Q3F1HPF)